netman/core/objects/switch_base.py

```diff
@@ -21,212 +21,212 @@
 class SwitchOperations(BackwardCompatibleSwitchOperations):
 
     def connect(self):
-        raise NotImplementedError()
+        raise NotImplementedError("'connect' is not implemented")
```

Comment:
what would you think of a decorator such as :

@wraps
def not_implemented(fn):
    raise NotImplementedError("'{}' is not implemented".format(fn.__name__))

Reply:
would be nice !